spec/models/email_alert_signup_spec.rb

```diff
@@ -0,0 +1,80 @@
+require "spec_helper"
+
+describe EmailAlertSignup do
+
+  let (:content_item) {
+    OpenStruct.new(
+      title: "Employment",
+      description: "People like getting paid",
+      details: OpenStruct.new(
+        tags: OpenStruct.new(
+          policy: ["employment"]
+        ),
+        breadcrumbs: [
+          OpenStruct.new(title: "Economy", link: "https://www.gov.uk/government/policies/economy"),
+          OpenStruct.new(title: "The UK", link: "https://www.gov.uk"),
+        ]
+      )
+    )
+  }
+
+  let (:email_alert_api) { double(:email_alert_api) }
+
+  before do
+
+    EmailAlertFrontend.register_service(:email_alert_api, email_alert_api)
+
+    allow(email_alert_api).to receive(:find_or_create_subscriber_list).and_return(OpenStruct.new("subscriber_list" => OpenStruct.new("subscription_url" => "http://govdelivery_signup_url")))
```

Comment:
Would these specs be less brittle if they used the pre-existing helpers that ship with `gds-api-adapters`? e.g. [GdsApi::TestHelpers::EmailAlertApi#email_alert_api_creates_subscriber_list](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/test_helpers/email_alert_api.rb#L27).